lib/examples/bootstrap/ansible/inventory/group_vars/node2/bootstrap_system_users.yml

```diff
@@ -0,0 +1,15 @@
+---
+# don't create my current account on hosts
+system_users__self: False
+
+# dedicated admin account to manage servers
+system_users__accounts:
+
+  - name: 'ansible-admin'
+    group: 'ansible-admin'
+    comment: 'Ansible admin'
+    # home path is generated automatically
+    shell: '{{ system_users__fact_self_shell }}'
```

Comment:
This will use the shell you have set on your account on Ansible Controller. Perhaps for the sake of consitency and not relying ton the dynamic user configuration, you should use `/bin/bash` here instead? Otherwise the users will wonder where does this shell come from.